app/assets/stylesheets/mortgage_calculator/layouts/_sticky.scss

```diff
@@ -44,4 +44,44 @@
   .borrowing-range {
     display: none;
   }
+
```

Comment:
A couple of questions for this file:

1. It's under layouts yet there aren't any `l-` prefixes being used, is that deliberate?
2. We've always been given the impression that within a particular file the filename and the style prefixes should be the same, e.g. given an `assets/stylesheets/layouts/grid.scss` file we should prefix all styles with `l-grid`. This isn't being done here, does a different rule apply?